.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: 🐛 Bug Report
+about: Template for reporting bugs
+title: ''
+labels: bug, new
+assignees: ''
+
+---
+
+## Environment
+
+What is the environment that the bug happens?
+What is the Rollups version?
+
+## Expected behavior
+
+What is expected to happen?
+
+## Actual behavior
+
+What is happening instead?
+
```

Comment:
IMHO the environment section should be the last. It is relevant for the bug report, but not so relevant as to be the first thing in it.

Reply:
I agree. I will put it just before possible solutions because this last one should be optional.